src/pcre2_compile.c

```diff
@@ -6520,16 +6520,18 @@ for (;; pptr++)
     case META_COND_RNUMBER:   /* (?(Rdigits) */
     case META_COND_NAME:      /* (?(name) or (?'name') or ?(<name>) */
     case META_COND_RNAME:     /* (?(R&name) - test for recursion */
-    case META_SCS_NAME:       /* Name of scan substring */
+    case META_CAPTURE_NAME:   /* Generic capture name */
     bravalue = OP_COND;
+
+    if (lengthptr != NULL)
```

Comment:
We have two code paths now, one for length computation and one for compilation.

Reply:
Nice! That seems good.